src/Variants.ts

```diff
@@ -255,11 +259,14 @@ const getMatchOrElseFn =
             const data = { ...instance }
             delete data[discriminant]
 
+            // eslint-disable-next-line @typescript-eslint/no-unsafe-return
             return typeof branch === "function" ? branch(data) : branch
         }
 
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-return
```

Comment:
❓ should we just disable `no-unsafe-return` in the whole file?

Reply:
Works for me :+1:

Reply:
https://github.com/taylorn16/fp-toolkit/pull/31/commits/2d8f2ee12a635a23ed6ce0052f29721077d0b928

I did a couple others, let me know what you think.

Reply:
Looks good!